desktop/cmp/button/AppMenuButton.js

```diff
@@ -17,74 +17,119 @@ export const [AppMenuButton, appMenuButton] = hoistCmp.withFactory({
     className: 'xh-app-menu',
 
     render(props) {
-        let {className, hideOptionsItem, hideFeedbackItem, hideThemeItem, hideAdminItem, hideImpersonateItem, hideLogoutItem, extraItems, ...rest} = props;
-        extraItems = extraItems ?
-            [...extraItems.map(m => menuItem(m)), menuDivider()] :
-            [];
+        let {className, hideOptionsItem, hideFeedbackItem, hideThemeItem, hideAdminItem, hideImpersonateItem, hideLogoutItem, extraItems, ...rest} = props,
```

Comment:
I think if we default `extraItems` to an empty array, we can skip the ternary logic below and always include them + menu divider (with the knowledge the divider will be removed if no items provided).